docs_app/content/navigation.json

```diff
@@ -30,7 +30,13 @@
         },
         {
           "url": "guide/operators",
```

Comment:
This is still clickable document, but it now has a child document:

<img width="612" alt="image" src="https://user-images.githubusercontent.com/28087049/218568265-b20963e1-d83d-447e-8b9f-187c7af73026.png">